src/app/shared/components/confirmation-modal-window/confirmation-modal-window.component.ts

```diff
@@ -29,7 +35,10 @@ export class ConfirmationModalWindowComponent implements OnInit {
     this.modalTitle = ModalConfirmationTitle[this.data.type];
     this.modalConfirmationText = ModalConfirmationText[this.data.type];
     if (this.data.property) {
-      this.modalConfirmationProperty = this.data.property;
+      this.modalConfirmationProperty = this.data.property.trim();
```

Comment:
Used `trim()` because some names that coming from DB has additional space at the end. Need to know that it is really should to be